docs/source/top_level_elements/language.rst

```diff
@@ -0,0 +1,172 @@
+    </language>
+    <language>
+        <languageTerm authority="iso639-2b" type="text">Italian</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1>
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/fre> ;
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/ita> .
+
+Present for 1 total in egypt
+
+https://digital.lib.utk.edu/collections/islandora/object/egypt:59/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm type="code" authority="iso639-2b">eng</languageTerm>
+    </language>
+    <language>
+        <languageTerm type="code" authority="iso639-2b">fre</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1>
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/eng> ;
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/fre> .
+
+language has language subelement with an @type of #text and value of "No linguistic content"
+--------------------------------------------------------------------------------------------
+
+https://digital.lib.utk.edu/collections/islandora/object/tdh:911/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm authority="iso639-2b" type="text">No linguistic content</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:language <http://id.loc.gov/vocabulary/iso639-2/zxx> ;
+
+language has language subelement with an @type = "code" and value of "zxx"
+--------------------------------------------------------------------------
+
+https://digital.lib.utk.edu/collections/islandora/object/tdh:911/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm type="code" authority="iso639-2b">zxx</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:language <http://id.loc.gov/vocabulary/iso639-2/zxx> ;
+
+
+
+language has language subelement without a stated @authority and a @code value of "fra"
+---------------------------------------------------------------------------------------
+
+Two questions here along with my proposed:
+
+1. How do we handle any that have no stated authority?
+2. Two volvoices objects use "fra". Is there a difference between fra and fre as codes?
+
+https://digital.lib.utk.edu/collections/islandora/object/volvoices:9928/datastream/MODS/vieww
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm type="code">fra</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:language <http://id.loc.gov/vocabulary/iso639-2/fre> ;
```

Comment:
You need a closing period instead of a semicolon.

Reply:
Addressed syntax issue in 6d1fdfe